services/movie.py

```diff
@@ -0,0 +1,41 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None
```

Comment:
It accepts Optional[list[int]] because default parameter is None
Fix in other places